src/apps/relay/netengine.c

```diff
@@ -45,6 +45,7 @@ static pthread_barrier_t barrier;
 
 ////////////// Auth Server ////////////////
 
+uint32_t relay_servers_in_use = 0;
```

Comment:
should this be a static variable?

Reply:
It is needed in src/apps/relay/ns_ioalib_engine_impl.c and is a "global var". So no unless there is a special collection of global vars somewhere else...